### DISCUSSION TITLE
Running different container processor types on M1 Mac

### DISCUSSION THREAD

**Discussion by dan-burt** [abiosoft/colima#558] [Q&A]
It can be started with either AMD64 or ARM64. I assume this applies for all running containers, dictating the processor type of that image, and therefore the Hub image it is pulled from?

What are the typical performance constraints of this? And people's typical split of the available compute resources when starting `colima`?

I only run the usual selection of 10-20 containers for managing media libraries, few development sites and other network management stuff. Is the M1 or M2 so highly powered it would appear just as fast?

**Comment by rfay** [comment]
If you want decent and predictable setup, you'll want to run with the same architecture as your machine. So if you're on a mac M1, use arm64.
